src/parsing/index.ts

```diff
@@ -118,3 +119,16 @@ export function parseGenericSql(queryBeforeCursor: string, queryAfterCursor: str
     let parser = genericAutocompleteParser as Parser;
     return parser.parseSql(queryBeforeCursor, queryAfterCursor);
 }
+
+export function parseGenericSqlWithoutCursor(queryBeforeCursor: string): ParseResult {
+    return parseGenericSql(queryBeforeCursor + ' ', '');
```

Comment:
It's good to add a comment - why are we adding space. Checkout how I did it in our repository.

This line is not very transparent.